framework/src/meshgenerators/GeneratedMeshGenerator.C

```diff
@@ -82,6 +82,8 @@ GeneratedMeshGenerator::validParams()
 
   params.addParam<std::vector<ExtraElementIDName>>("extra_element_integers",
                                                    "Names of extra element integers");
+  params.addParam<std::vector<std::vector<dof_id_type>>>("extra_element_integer_ids",
+                                                         "IDs of extra element integers");
```

Comment:
So this is a capability you want in this generator?

Reply:
No but this is the easiest way of testing the parser. Also this can be a nice capability for users to test things.

Reply:
Could you instead create a test generator object, a la something similar to the test object I created for testing remote element deletion, `AlwaysDeleteRemotesGeneratedMeshGenerator`?

Reply:
We should not add capabilities to production objects that won't or shouldn't be used in production

Reply:
If you think this parameter can/should be used in production, that's a good argument for keeping it. Making your life a little more convenient because you don't need to make a test object is not a good argument